app/helpers/filter-replace.js

```diff
@@ -7,6 +7,19 @@ var filters = {
     'Research Papers in Economics': 'RePEc'
 };
 
+/**
+ * @module ember-preprints
+ * @submodule helpers
+ */
+
+/**
+ * filterReplace helper. Replaces long provider names without messing with search filter logic
+ *
+ * @class filterReplace
+ * @param {String} filter Filter
```

Comment:
Can a `class` have `params`? Or does this need to be a `method`? (Applies to various places in these docs)

It appears that ember's builtin template helpers (like hash) use `@for`, which could be an option if we don't want to declare things as both classes *and* methods all over the place:
http://yui.github.io/yuidoc/syntax/index.html#for
https://github.com/emberjs/ember.js/blob/v2.11.0/packages/ember-glimmer/lib/helpers/hash.js#L31